enhancements/console/quick-starts.md

```diff
@@ -96,15 +98,17 @@ QuickStarts CR for [Explore Serverless](https://marvelapp.com/236ge4ig/screen/69
 apiVersion: console.openshift.io/v1
 kind: QuickStart
 metadata:
-  name: explore-serverless
+  name: explore-serverless~en
```

Comment:
I'd suggest just leaving the postfix off for English Quick Starts. Then we don't have to worry about renaming existing Quick Starts. (We should mention that explicitly below.)

Reply:
I agree. But should we also be accepting of the `en` postfix?

Reply:
i guess at the end of the day the name only matters when we reference quick starts in other fields and try to resolve to the the correct quick start. We should look for both with and without the `en` postfix.